data_structures/src/chain.rs

```diff
@@ -636,7 +636,8 @@ impl Hashable for PublicKey {
 #[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize, ProtobufConvert, Default)]
 #[protobuf_convert(pb = "witnet::Block_BlockHeader")]
 pub struct BlockHeader {
-    /// The block version number indicating the block validation rules
+    /// 32 bits for binary signaling new witnet protocol improvements
+    // TODO: Rename `version` to `signals` when we support migrations
     pub version: u32,
```

Comment:
Renaming is supported by both bincode and protobuf, but it would break the JSON-RPC API.